components/navigation/Navigation.js

```diff
@@ -55,7 +55,6 @@ export default function Navigation({ navigationData }) {
             <HamburgerImage
               src={icons.HAMBURGER}
               onClick={() => {
-                document.body.style.overflow = 'hidden';
```

Comment:
I would remove everywhere we added `document.body.style.overflow = 'visible';` too since it was meant to override this